### ISSUE TITLE
nmake install fails to install modules

### ISSUE THREAD

**Issue by GauthamBanasandra** [openssl/openssl#11695]
I'm trying to build OpenSSL on Windows 10 Pro. My clone of OpenSSL is at commit `47f387e98ebc9867ee7902a43d7bb35462175ce6`.
Here are the commands that I followed to build OpenSSL -
```
> git clone https://github.com/openssl/openssl.git
> cd openssl
> perl Configure VC-WIN64A --with-zlib-include=C:\PROGRA~1\Zlib\include --with-zlib-lib=C:\PROGRA~1\Zlib\bin\zlib1.dll --prefix=C:/OpenSSL --openssldir=C:/OpenSSL
> nmake
> nmake install
```
Here's the error that I'm getting -
```
*** Installing modules
Destination must be a directory at .\util\copy.pl line 53.
The filename, directory name, or volume label syntax is incorrect.
NMAKE : fatal error U1077: 'if' : return code '0x1'
Stop.
```
Please note the following -
1. `nmake` command executes successfully without any errors. The above error comes only when I run `nmake install`.
2. I've built zlib and put it in `C:\PROGRA~1\Zlib`
3. I'm using Strawberry Perl v5.30.2
4. I'm using Visual Studio 2019 x64 command prompt in Administrator mode.

**Comment by mattcaswell** [comment]
This is a dupe of issue #11623 which was fixed by commit 70411a50d (co-incidentally the commit that was commited immediately after the one you checked out).

BTW, I assume you realise you are running the dev branch (master)? If you need stability you need the branch for 1.1.1.

**Comment by GauthamBanasandra** [comment]
> If you need stability you need the branch for 1.1.1.

I didn't know that. Thanks @mattcaswell .
